src/api/index.tsx

```diff
@@ -51,7 +53,7 @@ export const callApi = async ({
 
     return await response.json();
   } catch (error) {
-    console.error(error);
+    log.error(error);
```

Comment:
This can probably be `log.debug(error)` (maybe with a string prefix explaining that `callApi` caught the error).